src/graphql/handler.rs

```diff
@@ -1,34 +1,37 @@
-use crate::database::pool::PgPool;
-use crate::graphql::manager::create_context;
-use crate::graphql::model::Schema;
-use actix_web::{web, Error, HttpResponse};
-use futures::Future;
-use juniper::http::GraphQLRequest;
-use std::sync::Arc;
-extern crate serde_json;
+use crate::cli_args::Opt;
+use crate::database::{db_connection, Pool};
+use crate::graphql::model::{Context, Schema};
 use crate::jwt::model::DecodedToken;
-use crate::serde::ser::Error as SerdeError;
 use crate::user::model::LoggedUser;
+use actix_web::{error, web, Error, HttpResponse};
+use juniper::http::graphiql::graphiql_source;
+use juniper::http::GraphQLRequest;
+use std::sync::Arc;
 
-pub fn graphql_handler(
+pub(super) async fn graphql(
```

Comment:
I think the `_handler` postfix was redundant with the file name already being `handler.rs`.